cms/tests/test_extensions.py

```diff
@@ -31,8 +31,9 @@ def test_register_extension(self):
         page_extension = self.get_page_extension_class()
 
         # register first time
-        extension_pool.register(page_extension)
+        register_extension_result = extension_pool.register(page_extension)
```

Comment:
As I said the test can't run because of the error. Before test runner even start finding tests, when django is trying to register the model into administration the error is raised.
This means that there is no need to change the tests (compared to code before my PR), because they already test if the models are registered as extensions. And if someone broke the functionality to use `register` as decorator instead of one test failing the whole test suite will not run.

Are you ok with this situation?